mongo/bulk_write.go

```diff
@@ -287,7 +291,7 @@ func (bw *bulkWrite) runUpdate(ctx context.Context, batch bulkWriteBatch) (opera
 		Session(bw.session).WriteConcern(bw.writeConcern).CommandMonitor(bw.collection.client.monitor).
 		ServerSelector(bw.selector).ClusterClock(bw.collection.client.clock).
 		Database(bw.collection.db.name).Collection(bw.collection.name).
-		Deployment(bw.collection.client.deployment).Crypt(bw.collection.client.crypt)
+		Deployment(bw.collection.client.deployment).Crypt(bw.collection.client.crypt).Hint(hasHint)
```

Comment:
This Hint method already exists on Deployment?  What does it do?

Reply:
@jyemin Can you explain what you mean? `Deployment` is an interface that does not have a `Hint` method.

Reply:
Mis-read it.